### PR TITLE
Add a networkdb API to get the members

### PR DESCRIPTION
**Pull request by sanimej**
For reporting the gossip cluster status in swarmkit libnetwork needs an API to fetch the gossip members.

Signed-off-by: Santhosh Manohar santhosh@docker.com

### CODE REVIEW

**Comment by aboch** [comment]
LGTM

File: agent.go [hunk 1]
```diff
@@ -41,6 +41,15 @@ type agent struct {
 	driverCancelFuncs map[string][]func()
 }
 
+func (c *controller) GossipClusterMembers() []string {
+	c.Lock()
+	defer c.Unlock()
+	if c.agentInitDone == nil {
```
Comment (mrjana) [inline agent.go]: Can you make this check for this `if c.agent != nil && c.agentInitDone == nil` instead? I know this code will work correctly today based on the current implementation but to make this check robust and decoupled for future code changes we should only try to get `Members` only if `c.agent` is not nil.

**Comment by dongluochen** [comment]
Are we moving forward with this approach?

**Comment by sanimej** [comment]
Not for 1.13. The idea when I pushed the PR was to show the gossip cluster count in `node inspect` output. But there is a concern with adding dynamic operational state into raft. Instead what I have done for 1.13 is to show the gossip member list per network in `network inspect`

**Comment by fcrisciani** [comment]
not anymore valid

**Pull request closed without merging**